src/Security/Authorization/Core/src/AuthorizationServiceExtensions.cs

```diff
@@ -21,8 +21,8 @@ public static class AuthorizationServiceExtensions
     /// <param name="resource">The resource to evaluate the policy against.</param>
     /// <param name="requirement">The requirement to evaluate the policy against.</param>
     /// <returns>
-    /// A flag indicating whether requirement evaluation has succeeded or failed.
-    /// This value is <c>true</c> when the user fulfills the policy, otherwise <c>false</c>.
+    /// A <see cref="Task{TResult}"/> that contains an <see cref="AuthorizationResult"/> indicating whether requirement evaluation has succeeded or failed.
+    /// The result's <see cref="AuthorizationResult.Succeeded"/> property is <c>true</c> when the user fulfills the policy, otherwise <c>false</c>.
```

Comment:
```suggestion
    /// The result's <see cref="AuthorizationResult.Succeeded"/> property is <c>true</c> when the user fulfills the policy; otherwise <c>false</c>.
```